apps/prairielearn/public/stylesheets/home.css

```diff
@@ -1,95 +0,0 @@
-:root {
-  --flare-width-before: 8rem;
-  --flare-width: calc(2 * var(--flare-width-before));
-  --flare-width-after: calc(3 * var(--flare-width-before));
-}
-
-.tiny {
-  font-size: 70%;
-}
```

Comment:
This was the only block that had a use in the page after removing what was no longer necessary. I opted for adding it as an element fixed style instead, so that this file could be removed.